soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -79,6 +79,8 @@ object SoQLFunctions {
   val UnaryPlus = Function("unary +", SpecialFunctions.Operator("+"), Map("a"->NumLike), Seq(VariableType("a")), Seq.empty, VariableType("a"))
   val UnaryMinus = Function("unary -", SpecialFunctions.Operator("-"), Map("a"->NumLike), Seq(VariableType("a")), Seq.empty, VariableType("a"))
 
+  val Magnitude = Function("magnitude", FunctionName("magnitude"), Map("a"->NumLike, "b"->NumLike), Seq(VariableType("a"), VariableType("b")), Seq.empty, VariableType("b"))
```

Comment:
The result is always a positive integer. Similarly the base argument is always a positive integer.